examples/Deploying-multi-stage-RecSys/README.md

```diff
@@ -18,18 +18,15 @@ You can run the example notebooks by [installing NVTabular](https://github.com/N
 
 To run the example notebooks using Docker containers, do the following:
 
-1. Pull the container by running the following command:
+1. Once you pull the inference container, launch it by running the following command:
    ```
-   docker run --runtime=nvidia --rm -it -p 8888:8888 -p 8797:8787 -p 8796:8786 --ipc=host --cap-add SYS_PTRACE <docker container> /bin/bash
+   docker run -it --gpus all -p 8000:8000 -p 8001:8001 -p 8002:8002 -p 8888:8888 -v <path to your data>:/workspace/data/ --ipc=host <docker container> /bin/bash
    ```
 
-   **NOTE**: If you are running on Docker version 19 and higher, change ```--runtime=nvidia``` to ```--gpus all```.
+   **NOTE**: If you are running on Docker version 19 and higher, change ```--runtime=nvidia``` to ``````.
```

Comment:
This note does note make sense. It should be `If you are running on Docker version 18 or lower, change --gpus all to --runtime=nvidia`. I would remove the note at all and just assume, that we are useing version 19 or higher.